mmv1/third_party/terraform/services/bigquery/resource_bigquery_table.go.erb

```diff
@@ -1809,6 +1840,18 @@ func resourceBigQueryTableDelete(d *schema.ResourceData, meta interface{}) error
 	if d.Get("deletion_protection").(bool) {
 		return fmt.Errorf("cannot destroy instance without setting deletion_protection=false and running `terraform apply`")
 	}
+	<% unless version == 'ga' -%>
+	if v, ok := d.GetOk("resource_tags"); ok {
```

Comment:
Does the API have this check / does it take a long time to report a failure? Otherwise, if the API will reject the call there's not much value in doing this clientside first, since this will save 1s / one API call AIUI.

I'm a little hesitant about just checking state here- Terraform will sometimes short-circuit delete calls so Terraform may have drifted from the real config. I'm not sure we ever drift from state though, it's been a bit since I knew all the rules.

Reply:
The deletion will go through actually, but what I discovered while debugging the VCR test failure is that the tag value bindings will remain if a table is deleted while having resource tags specified, and that makes it impossible to delete the tag value resource, failing at "it's still has an association". As a follow-up I'll reach out to our API team to discuss if the behavior should be fixed on the API-level.